kubernetes/spack/gitlab-spack-io/web/deployment.yaml

```diff
@@ -129,6 +127,21 @@ spec:
             secretKeyRef:
               name: gitlab-secrets
               key: gitlab-runner-token
+        - name: GITLAB_POST_RECONFIGURE_SCRIPT
+          value: >
+            src_file='/opt/gitlab/embedded/service/gitlab-rails/lib/gitlab'
+            src_file="$src_file/ci/pipeline/seed/build.rb"
+
+            sed -i -e 's/LOW_NEEDS_LIMIT = [0-9]\+/LOW_NEEDS_LIMIT = 150/g'   \
+                   -e 's/HARD_NEEDS_LIMIT = [1-9]\+/HARD_NEEDS_LIMIT = 200/g' \
+                "$src_file"
+
+            (
+              echo 'Feature.disable(:ci_dag_limit_needs);'
```

Comment:
Can we also disable the .gitlab-ci.yaml size limit feature flag while we're in here?

```
Feature.disable(:ci_yaml_limit_size)
```